server/src/optima/dbmodels.py

```diff
@@ -46,7 +46,9 @@ class ProjectDb(db.Model):
                                 uselist=False)
     project_data = db.relationship('ProjectDataDb', backref='projects',
                                 uselist=False)
-    creation_time = db.Column(db.DateTime(timezone=True), server_default=text('now()'))
+    creation_time = db.Column(db.DateTime(timezone=True), default=db.func.now())
```

Comment:
Project is updated when you e.g. upload a new json.

Model is updated when you run a new calculation and save the result.